server/repository/src/db_diesel/item_variant/item_variant_row.rs

```diff
@@ -25,13 +27,17 @@ allow_tables_to_appear_in_same_query!(item_variant, item);
     Clone, Queryable, Insertable, AsChangeset, Debug, PartialEq, Default, Serialize, Deserialize,
 )]
 #[diesel(table_name = item_variant)]
+#[diesel(treat_none_as_null = true)]
 pub struct ItemVariantRow {
     pub id: String,
     pub name: String,
     pub item_link_id: String,
     pub cold_storage_type_id: Option<String>,
     pub manufacturer_link_id: Option<String>,
     pub deleted_datetime: Option<chrono::NaiveDateTime>,
+    #[serde(default)]
+    pub doses_per_unit: i32,
```

Comment:
Copying item pattern to save non-vaccine items as 0... Optional better since currently our frontend makes all non-optionals null/undefined anyways 🤷🏻